lib/Onyx.js

```diff
@@ -62,6 +63,18 @@ let onClearCallback = null;
 let batchUpdatesPromise = null;
 let batchUpdatesQueue = [];
 
+/**
+ * Sends an action to DevTools extension
+ *
+ * @param {string} method - Onyx method from METHOD
+ * @param {string} key - Onyx key that was changed
+ * @param {any} value - contains the change that was made by the method
+ * @param {any} mergedValue - (optional) value that was written in the storage after a merge method was executed.
+ */
+function sendActionToDevTools(method, key, value, mergedValue = undefined) {
+    DevTools.registerAction(utils.formatActionName(method, key), value, key ? {[key]: mergedValue || value} : value);
```

Comment:
Should we _maybe_ wrap this in a try/catch _just_ in case? I am guessing it is not possible it would throw, but if somehow we passed a bad value I think we don't care because it's a devtool. Not sure if it makes sense. Maybe it's not possible 😄

Reply:
@marcaaron I can add a try/catch inside of `Devtools.registerAction` to prevent any errors from spreading to Onyx!

Reply:
@marcaaron Just added error handling on all functions that communicate directly with the extension.